.github/workflows/python-package.yml

```diff
@@ -30,14 +30,23 @@ jobs:
         python -m pip install --upgrade pip
         pip install .
         python -c 'import acoular'
+    - name: Test ubuntu with unittest *nix
+      run: |
+        cat /proc/cpuinfo
+        chmod +x ../acoular/acoular/tests/run_tests.sh
+        cd ../acoular/acoular/tests/
+        bash run_tests.sh
+      if: ${{ matrix.os == 'ubuntu-latest'}} 
     - name: Test with unittest *nix
```

Comment:
Isn't this macos ?

Reply:
That's obviously ubuntu.  Why should it be macos? See the latest test action: https://github.com/acoular/acoular/actions/runs/8736226608

Reply:
because then we have already Test ubuntu with unittest *nix in l 33
and now it is Test with unittest *nix in l 40 when actually macos-latest is requested

I think the '*nix' was a label for both ubuntu and macos and now we have ubuntu and macos as different actions. So they should have correct labels

Reply:
What is this '*nix' label for?

Reply:
[Li,U]nix